custom/ewsghana/reports/__init__.py

```diff
@@ -162,3 +165,29 @@ def is_reporting_type(self):
             if not location_type.administrative
         ]
         return sql_location.location_type in reporting_types
+
+    @property
+    def export_table(self):
+        r = self.report_context['reports'][0]['report_table']
+        return [self._export_table(r['title'], r['headers'], r['rows'])]
+
+    def _export_table(self, export_sheet_name, headers, formatted_rows, total_row=None):
```

Comment:
@czue I've just move it to super class because of every multireport will be exportable because of facility report page.